pkgs/top-level/all-packages.nix

```diff
@@ -23401,7 +23401,9 @@ in
 
   thunderbird = thunderbird-78;
   thunderbird-78 = callPackage ../applications/networking/mailreaders/thunderbird {
-    inherit (rustPackages_1_44) cargo rustc;
+    # Using older Rust for workaround:
+    # https://bugzilla.mozilla.org/show_bug.cgi?id=1663715
+    inherit (rustPackages_1_45) cargo rustc;
```

Comment:
Well, default version of rust will surely remain constant on stable branches (and is 1.45 now) and the diff looks a little self-contradictory here ("using older" upgrade)... but I actually don't see any problem in keeping this PR as it is.